package.json

```diff
@@ -29,7 +29,7 @@
     "@oclif/config": "^1.17.0",
     "@salesforce/command": "^3.1.0",
     "@salesforce/core": "^2.18.0",
-    "@salesforce/lwc-dev-mobile-core": "beta",
+    "@salesforce/lwc-dev-mobile-core": "^1.0.0-beta2",
```

Comment:
We need to bump the version of this package too, to beta2.

Reply:
🤦‍♂️